cypress/plugins/index.js

```diff
@@ -12,10 +12,24 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv").config({ path: ".env.local" });
+
 /**
  * @type {Cypress.PluginConfig}
  */
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  setTestEnvVarsFromSystem(config);
+
+  return config;
+};
+
+const setTestEnvVarsFromSystem = (config) => {
+  config.env.WEBAPP_CLIENT_ID = process.env.WEBAPP_CLIENT_ID;
+  config.env.WEBAPP_CLIENT_SECRET = process.env.WEBAPP_CLIENT_SECRET;
+  config.env.AAD_TENANT_ID = process.env.AAD_TENANT_ID;
+  config.env.AAD_API_ID = process.env.AAD_API_ID;
+  config.env.API_URL = process.env.API_URL;
```

Comment:
Used to make env vars both from system and in a `.env.local` file available to Cypress, avoiding having to duplicate environments. Otherwise would have to set WEBAPP_CLIENT_ID *and* CYPRESS_WEBAPP_CLIENT_ID etc.